modules/host/upnp.go

```diff
@@ -69,12 +78,14 @@ func (h *Host) managedLearnHostname() {
 	// has a storage obligation. If the host is not accepting contracts and has
 	// no open contracts, there is no reason to notify anyone that the host's
 	// address has changed.
-	if h.settings.AcceptingContracts || h.financialMetrics.ContractCount > 0 {
-		err = h.announce(autoAddress)
+	if hAcc || hCC > 0 {
```

Comment:
laziness is a disease.

If the variable is used more than 3-4 lines from where it's declared, I'll change it.